src/test/java/teammates/test/cases/logic/EmailGeneratorTest.java

```diff
@@ -77,7 +92,19 @@ public void testGenerateFeedbackSessionEmails() throws IOException {
         subject = String.format(EmailType.FEEDBACK_SESSION_REMINDER.getSubject(),
                                 course.getName(), session.getFeedbackSessionName());
         
-        verifyEmail(emails.get(1), instructor1.email, subject, "/sessionReminderEmailForInstructor.html");
+        hasStudent1ReceivedEmail = false;
+        hasInstructor1ReceivedEmail = false;
+        for (EmailWrapper email : emails) {
+            if (email.getRecipient().equals(student1.email)) {
+                verifyEmail(email, student1.email, subject, "/sessionReminderEmailForStudent.html");
+                hasStudent1ReceivedEmail = true;
+            } else if (email.getRecipient().equals(instructor1.email)
+                       && email.getContent().contains("The email below has been sent to students of course:")) {
```

Comment:
This is so that only the email copy sent to instructors are tested (the email meant as reminder for instructor is not tested yet). Related to #5697.